src/format/xVecLibBoomerangFormatV0_1/index.tsx

```diff
@@ -0,0 +1,138 @@
+  }
+
+  getPathPoints(path: Path): PointCalculationResult {
+    const result = getPathPoints(path, new Quantity(this.gc.pointDensity, this.gc.uol));
+    return result;
+  }
+
+  convertFromFormat(oldFormat: Format, oldPaths: Path[]): Path[] {
+    return convertFormat(this, oldFormat, oldPaths);
+  }
+
+  importPathsFromFile(buffer: ArrayBuffer): Path[] {
+    throw new Error("Unable to import paths from this format, try other formats?");
+  }
+
+  exportCode(): string {
+    const { app } = getAppStores();
+    let rtn = "";
+    let arr = [];
+
+    const gc = app.gc as GeneralConfigImpl;
+
+    const path = app.interestedPath();
+    if (path === undefined) throw new Error("No path to export");
+    if (path.segments.length === 0) throw new Error("No segment to export");
+
+    const uc = new UnitConverter(this.gc.uol, UnitOfLength.Inch);
+    const points = path.segments;
+    if (points.length > 0) {
+      for (const point of points) {
+        if (path.segments.indexOf(point) === 0) {
+          console.log(point.first.heading);
+          let tmpp = point.first.heading > 180 ? point.first.heading - 360 : point.first.heading;
+
+          rtn += `${gc.chassisName}.setPos(${Math.round(point.first.x)}, ${Math.round(point.first.y)},${tmpp});\n`;
+          rtn += `${gc.chassisName}.printCoords();\n`;
+        }
+        let pnt = 0;
+        let last = path.controls.at(path.controls.length - 1);
+        let dis;
+        if (last != null) {
+          dis = path.controls.at(0)?.distance(last);
+          if (point.isCubic() && dis != null) {
+            let poin =
+              point.controls[1].distance(point.first) > point.controls[2].distance(point.last)
+                ? point.controls[1]
+                : point.controls[2];
+            if (point.first.heading === 0) {
+              point.first.heading =
+                Math.atan2(point.first.x - poin.x, point.first.y - poin.y) * (180 / 3.14159265358979323846);
+            }
+            if (point.last.heading === 0) {
+              point.last.heading =
+                Math.atan2(point.last.x - poin.x, point.last.y - poin.y) * (180 / 3.14159265358979323846);
+            }
+            if (path.segments.indexOf(point) === path.segments.length - 1) {
+              pnt = (point.first.x - poin.x) / (dis * Math.sin(point.first.heading));
+            } else {
+              pnt = (point.last.x - poin.x) / (dis * Math.sin(point.last.heading));
+            }
+            console.log(pnt);
+          }
+        }
+        let tmpp = point.last.heading > 180 ? point.last.heading - 360 : point.last.heading;
+        if (pnt > 1) {
+          pnt /= 10;
+        }
+        arr.push([gc.chassisName, uc.fromAtoB(point.last.x).toUser(), uc.fromAtoB(point.last.y).toUser(), tmpp, pnt]);
+      }
+    }
+    for (const s of arr) {
+      rtn += `${s[0]}.moveToBoom( ${s[1]}, ${s[2]}, ${s[3]}, ${s[4]},${gc.movementTimeout / 1000});\n`;
```

Comment:
I think it is better for users to input `Movement Timeout` in milliseconds on the user interface, and remove `/ 1000` here in the function.